utils/bloom_calculations.hh

```diff
@@ -123,6 +123,19 @@ namespace bloom_calculations {
         }
         return std::min(probs.size() - 1, size_t(v));
     }
+
+    /**
+     * Retrieves the minimum supported bloom_filter_fp_chance value
+     * if compute_bloom_spec() above is attempted with bloom_filter_fp_chance
+     * lower than this, it will throw an unsupported_operation_exception.
+     */
+    inline double min_supported_bloom_filter_fp_chance()
+    {
```

Comment:
Ah, sorry, I should've caught it in the previous round of review. According to our coding style, the opening curly brace should be included in the previous line.

Reply:
Oops, maybe I inherited the bad style from the Java code I looked at ;-)
Pushed a new version.